fe/fe-core/src/main/java/org/apache/doris/job/base/AbstractJob.java

```diff
@@ -54,35 +54,73 @@
 @Log4j2
 public abstract class AbstractJob<T extends AbstractTask, C> implements Job<T, C>, Writable {
 
-    @SerializedName(value = "jid")
+    @SerializedName(value = "jobId")
```

Comment:
Do not change it.
We suggest to use abbr in future, to minimize the edit log size